pingcap/dm
# checker: allow to ignore some checking items

### What problem does this PR solve? 
allow to ignore some checking items

### What is changed and how it works?
add a `IgnoreCheckingItems` configuration, and use it to filter checking items in `checker`

### Check List 

Tests 

 - Unit test
 - Manual test (add detailed scripts or steps below)
    - set up a dm cluster
    - start a config by task
    - check output in log

Code changes
 - Has exported variable/fields change
